admin/validate_herbert.m

```diff
@@ -65,7 +65,7 @@
 initial_warn_state = warning();
 warning('off', 'MATLAB:class:DestructorError');
 % Generate full test paths to unit tests:
-rootpath = herbert_root();
+rootpath = fileparts(fileparts(which('herbert_init')));
```

Comment:
why is this function changed?

Reply:
See https://github.com/pace-neutrons/Herbert/pull/108/commits/cdd89474348a9e0d30fea8c93632fc7f555bda0d.

Tests were failing because of this . Maybe it needs its own issue/PR, but it's a very small change

Reply:
Isn't the solution to move `herbert_root` somewhere that is always on the path? That function was created to remove the impenetrably complex `fileparts(fileparts(which(...)))` call

Reply:
you might need to separate the `dps` piece out as that only does make sense in the test code.

Reply:
Happy for that restructure to be a new PR since we've added code code duplication in "now working" code.